readers/mongodb/messages.go

```diff
@@ -82,9 +82,6 @@ func (repo mongoRepository) ReadAll(chanID string, offset, limit uint64, query m
 	if err != nil {
 		return readers.MessagesPage{}, errors.Wrap(errReadMessages, err)
 	}
-	if total < 0 {
```

Comment:
It's not addressed in PR, not sure if it's accidentally removed?

Reply:
It's removed because `total` is `uint` type, so this can never be true.